code/hud/hudshield.h

```diff
@@ -15,9 +15,16 @@
 #include "globalincs/pstypes.h"
 #include "hud/hud.h"
 
-#define SHIELD_HIT_DURATION	1400	// time a shield quadrant flashes after being hit
+#define SHIELD_HIT_FLASH_DURATION	1400	// time a shield quadrant flashes after being hit
 #define SHIELD_FLASH_INTERVAL	200	// time between shield quadrant flashes
 
+enum ShieldGaugeType {
```

Comment:
This really should be an `enum class ShieldGaugeType : uint8_t` instead.

You can still use the last enum to get the count, you just need to use a static cast like this:
```static_cast<uint8_t>(ShieldGaugeType::num_shield_gauge_types)``` instead

Reply:
Per discussion, leaving this as is for now.